### PR TITLE
Clean up strings

### PR DESCRIPTION
**Pull request by levbishop**
Cleaning up multiline strings.

- A few places strings were broken into more lines than necessary.
- Removed f"" from strings where no substitution is done
- ~~Added f"" to a few strings which intended to do substitution but were missing the prefix~~
- Parenthenthesized multi-line strings in the middle of function calls, for improved readabililty
- Also improved a couple python2-isms i noticed along the way

### CODE REVIEW

File: qiskit/transpiler/preset_passmanagers/level0.py [hunk 1]
```diff
@@ -145,8 +145,10 @@ def _swap_condition(property_set):
     elif routing_method == "none":
         _swap += [
             Error(
-                msg="No routing method selected, but circuit is not routed to device. "
-                "CheckMap Error: {check_map_msg}",
+                msg=(
+                    "No routing method selected, but circuit is not routed to device. "
+                    "CheckMap Error: {check_map_msg}"
```
Comment (jakelishman) [inline qiskit/transpiler/preset_passmanagers/level0.py:150]:
```suggestion
                    f"CheckMap Error: {check_map_msg}"
```

File: qiskit/transpiler/preset_passmanagers/level1.py [hunk 2]
```diff
@@ -164,8 +164,10 @@ def _swap_condition(property_set):
     elif routing_method == "none":
         _swap += [
             Error(
-                msg="No routing method selected, but circuit is not routed to device. "
-                "CheckMap Error: {check_map_msg}",
+                msg=(
+                    "No routing method selected, but circuit is not routed to device. "
+                    "CheckMap Error: {check_map_msg}"
```
Comment (jakelishman) [inline qiskit/transpiler/preset_passmanagers/level1.py:169]:
```suggestion
                    f"CheckMap Error: {check_map_msg}"
```

File: qiskit/transpiler/preset_passmanagers/level2.py [hunk 3]
```diff
@@ -198,8 +198,10 @@ def _swap_condition(property_set):
     elif routing_method == "none":
         _swap += [
             Error(
-                msg="No routing method selected, but circuit is not routed to device. "
-                "CheckMap Error: {check_map_msg}",
+                msg=(
+                    "No routing method selected, but circuit is not routed to device. "
+                    "CheckMap Error: {check_map_msg}"
```
Comment (jakelishman) [inline qiskit/transpiler/preset_passmanagers/level2.py:203]:
```suggestion
                    f"CheckMap Error: {check_map_msg}"
```

File: qiskit/transpiler/preset_passmanagers/level3.py [hunk 4]
```diff
@@ -201,8 +201,10 @@ def _swap_condition(property_set):
     elif routing_method == "none":
         _swap += [
             Error(
-                msg="No routing method selected, but circuit is not routed to device. "
-                "CheckMap Error: {check_map_msg}",
+                msg=(
+                    "No routing method selected, but circuit is not routed to device. "
+                    "CheckMap Error: {check_map_msg}"
```
Comment (jakelishman) [inline qiskit/transpiler/preset_passmanagers/level3.py:206]:
```suggestion
                    f"CheckMap Error: {check_map_msg}"
```

File: qiskit/transpiler/passes/basis/unroller.py [hunk 5]
```diff
@@ -93,9 +92,8 @@ def run(self, dag):
                     rule = rule[0][0].definition.data
                 except (TypeError, AttributeError) as err:
                     raise QiskitError(
-                        f"Error decomposing node of instruction '{node.name}': "
-                        f"{err}. Unable to define instruction '{rule[0][0].name}'"
-                        f" in the given basis."
+                        f"Error decomposing node of instruction '{node.name}': {err}. "
+                        f"Unable to define instruction '{rule[0][0].name}'in the given basis."
```
Comment (jakelishman) [inline qiskit/transpiler/passes/basis/unroller.py:96]:
```suggestion
                        f"Unable to define instruction '{rule[0][0].name}' in the given basis."
```

File: qiskit/transpiler/preset_passmanagers/level0.py (hunk 1 above)
Comment (levbishop) [reply qiskit/transpiler/preset_passmanagers/level0.py]: Seems like there's a few places where f prefix was initially missing. I'll try to follow up to this PR with a mechanical check for that situation.

**Comment by levbishop** [comment]
Thanks for the careful review @jakelishman I'll make a follow up PR to look for other places where f prefix may have been missing

**Review by jakelishman — APPROVED** [review]
Looks good to me - I'm always in favour of simplifying these things.

File: qiskit/transpiler/preset_passmanagers/level0.py (hunk 1 above)
Comment (levbishop) [reply qiskit/transpiler/preset_passmanagers/level0.py]: This was actually correct and `Error` pass does it's own substitution

**Review by jakelishman — APPROVED** [review]

**Review by jakelishman — APPROVED** [review]
Something weird happened with coveralls in the middle - it reported a big dip in coverage in unrelated files.  No idea what happened, but maybe it's something to keep an eye on in the future.

**Pull request merged**